.github/workflows/test.yml

```diff
@@ -0,0 +1,18 @@
+name: Test
```

Comment:
TLDR; this would be a great addition *except*, It's already happening/duplicative. 

the auto-generated [build action](https://github.com/opspec-pkgs/github.release.create/blob/a4e69c877313dfac22a8f57a1a596c169afabed7/.github/workflows/build.yml#L3) invokes the auto-generated [build op](https://github.com/opspec-pkgs/github.release.create/blob/a4e69c877313dfac22a8f57a1a596c169afabed7/.opspec/build/op.yml#L1) from bootstrap includes running the auto-generated [test op](https://github.com/opspec-pkgs/github.release.create/blob/a4e69c877313dfac22a8f57a1a596c169afabed7/.opspec/test/op.yml#L1) which runs the [opctl.op.validate op](https://github.com/opspec-pkgs/github.release.create/blob/a4e69c877313dfac22a8f57a1a596c169afabed7/.opspec/test/op.yml#L5)

Reply:
Removed